src/commands/org/login/web.ts

```diff
@@ -116,9 +116,13 @@ export default class LoginWeb extends AuthBaseCommand<AuthFields> {
       return fields;
     } catch (err) {
       const error = err as Error;
-      Logger.childFromRoot('auth').debug(error);
+      Logger.childFromRoot('LoginWebCommand').debug(error);
       if (error.name === 'AuthCodeExchangeError') {
-        throw new SfError(messages.getMessage('invalidClientId', [error.message]));
+        const authError = new SfError(messages.getMessage('invalidClientId', [error.message]));
```

Comment:
the 4th or 5th property of SfError (should been an object after 2!) allow you pass in the thrown error, rather than meddling with the stack.

Can you see if that produces uses output for the user, and if it *doesn't* let's fix SfError so that it does.

Also, I think the stack only appears in dev mode (ex: bin/dev) and maybe `--json`.